### PR TITLE
Dutch translation file added.

### PR DESCRIPTION
**Pull request by mhartgring**
I like your project so I have created a Dutch translation file for you.

(I hope I did this right, it has been a while since I created a PR)

### CODE REVIEW

File: client/src/lang/translations/nl.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  "username-label": "Gebruikersnaam",
+  "password-label": "Wachtwoord",
+  "confirm-password-label": "Bevestig wachtwoord",
+  "login-button": "Login",
+  "registration-successful-please-login": "Registratie geslaagd. Log in!",
+  "register-header": "Registeer",
+  "register-button": "Registeer",
+  "not-yet-saved": "Wijzigingen nog niet opgeslagen!",
+  "calendar-locale": "nl",
+  "modal-history-title": "Geschiedens van geselecteerde dag",
```
Comment (Stokkie64) [inline client/src/lang/translations/nl.json:37]: Geschiedenis van de geselecteerde dag

File: client/src/lang/translations/nl.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  "use-selected-history-as-default": "Stel geselecteerde geschiedenisversie in als nieuwste tekst?",
+  "yes": "Ja",
+  "no-history-available-yet": "Er is nog geen geschiedenis aanwezig!",
+  "export-started-1": "Zip-bestand wordt gemaakt. Dit kan enkele minuten duren, afhankelijk van de grootte.",
+  "export-started-2": "BELANGRIJK: Herlaad de pagina NIET tijdens dit proces!",
+  "close": "Sluiten",
+  "download": "Downloaden",
+  "modal-update-available-header": "Nieuwere versie van DailyTxT beschikbaar!",
+  "update-available-client-version": "Huidig geinstalleerde versie",
```
Comment (Stokkie64) [inline client/src/lang/translations/nl.json:46]: geïnstalleerde

File: client/src/lang/translations/nl.json [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  "update-available-recent-version": "Nieuwste versie",
+  "error-uploading-file": "Probleem bij uploaden! Vermoedelijk is het bestand te groot.",
+  "import-data-header": "Opnieuw importeren van data",
+  "import-data-text-1": "Upload progressie:",
+  "import-data-text-2": "Encryptie en importeren van gegevens",
+  "import-successful": "Importeren gelukt",
+  "backup-codes": "Back-up codes",
+  "create-backup-codes": "Genereer back-up codes",
+  "create-new-backup-codes": "Genereer nieuwe back-up codes",
+  "backup-codes-description-1": "Back-up codes werken als eenmalige wachtwoorden. Ze kunnen altijd worden gebruikt in plaats van het normale wachtwoord (bijvoorbeeld als je het vergeten bent), maar slechts één keer.",
```
Comment (Stokkie64) [inline client/src/lang/translations/nl.json:56]: Back-up codes werken als eenmalige wachtwoorden. Je kunt ze altijd gebruiken in plaats van je gewone wachtwoord (bijvoorbeeld als je het bent vergeten), maar slechts één keer.

File: client/src/lang/translations/nl.json [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  "backup-codes-description-2": "Als je je wachtwoord wijzigt, worden alle back-up codes verwijderd en moet je nieuwe aanmaken.",
+  "attention": "Attentie",
+  "backup-codes-not-successful": "Fout bij het aanmaken van nieuwe back-up codes.",
+  "backup-codes-created-1": "Er zijn nieuwe back-up codes gemaakt.",
+  "backup-codes-created-2": "Noteer de back-up codes NU. Na het sluiten van dit venster zul je ze niet meer kunnen bekijken, maar je kunt altijd nieuwe aanmaken.",
+  "backup-codes-deleted": "Back-up codes zijn verwijderd. Je moet nieuwe aanmaken!",
+  "copy-to-clipboard-successful": "Back-up codes succesvol gecopieerd",
```
Comment (Stokkie64) [inline client/src/lang/translations/nl.json:63]: gekopieerd

File: client/src/lang/translations/nl.json [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  "update-installed-reload": "Nieuwe update is geïnstalleerd. Laad de pagina opnieuw!",
+  "remove-day": "Verwijder alle logs en bestanden van deze dag.",
+  "modal-remove-day-header": "Alle gegevens van deze dag verwijderen?",
+  "modal-remove-day-text": "Wilt je alle logs en bestanden van {0} permanent verwijderen?",
+  "settings": "Instellingen",
+  "close-settings": "Sluit instellingen",
+  "templates": "Sjablonen",
+  "export-data-description": "Maakt een ZIP-bestand om te downloaden dat alle teksten en geüploade bestanden bevat. De inhoud is <b>ontsleuteld</b>",
+  "create-export": "Export maken",
+  "import-data-description": "Selecteer je (eerder geëxporteerde) ZIP-bestand om de gegevens te uploaden. De teksten in het bestand worden behandeld als de nieuwste items. Het importeren kan een paar minuten duren (afhankelijk van de grootte).",
```
Comment (Stokkie64) [inline client/src/lang/translations/nl.json:75]: Selecteer je (eerder geëxporteerde) ZIP-bestand om gegevens te uploaden. De teksten worden als nieuwste items behandeld. Importeren kan enkele minuten duren, afhankelijk van de bestandsgrootte.

**Review by Stokkie64 — CHANGES_REQUESTED** [review]
Thanks so much for your PR! It's really appreciated. 
I've added some small feedback suggestions. Mostly small things or subjective style points. I leave it up to you which you want to take with. :)

**Comment by mhartgring** [comment]
Good eye, I rushed into making this. I'll update the PR asap.

**Review by ArjenMarcoSingels — APPROVED** [review]
Akkoord, Happy with the language. I leave it to @PhiTux to merge this if he's happy :)

**Review by Stokkie64 — APPROVED** [review]
Approved, @PhiTux I approved with my incorrect github account. Are you able to remove that approval?

**Comment by PhiTux** [comment]
Hi, 
thanks very much to everybody involved for the translation! 
I'm not exactly sure right now, if I will "just" try to update dailyTxT within the next days and integrate you translation into this, or if I should start to rewrite the whole thing. 
There are several new ideas now in the issues-section, that I would like to address. But I would then combine this with a rewrite...
Well, let's see...

**Comment by PhiTux** [comment]
@Stokkie64 I'm not sure how I could remove you other approval. You mean the one right in front of the second approval, where you wrote this, right?

**Comment by Stokkie64** [comment]
I wouldn't stress about removing it but Thanks.

**Comment by PhiTux** [comment]
Just for your interest: I'm planing to start a rewrite of dailytxt within the next days. I'll try to address most of the open issues within this step. The dutch translation will be included into this (although there will surely be several new text-snippets which need to be translated...)

**Pull request merged**